api/src/main/java/jakarta/persistence/Column.java

```diff
@@ -186,4 +186,15 @@
      * @since 3.2
      */
     String comment() default "";
+
+    /**
+     * Determines if this column is validated as part of the
+     * schema {@linkplain SchemaManager#validate validation}
+     * process. By default, the column is validated. If the
+     * column should be excluded from validation, specify
+     * {@code validated=false}.
```

Comment:
There is an important ambiguity here. What does "excluded from validation" mean precisely? Does it mean we check that the column exists but perform no further validation beyond that? Or does it mean we don't even validate that it exists in the table? 🤔